views/index.erb

```diff
@@ -1,41 +1,150 @@
-    <h3>Stats</h3>
-    <!-- TODO - Set the contents of this span with JavaScript -->
-    <span id="word-count">0</span> words found.
+  <div class="section-two">
+    <div class="title">
+      <h2><%=@profile.keys[0].capitalize%></h2>
+        <div class="content">
+          <h3 class ="profile-value"><%=@profile["about"] %></h3>
+        </div>
+    </div>
+
+    <div class="info-container">
+      <div id="name" class="basic-info">
+        <h2><%=@profile.keys[5].capitalize%></h2>
+        <h3 class ="profile-value"><%=@profile["first_name"] %> <%=@profile["last_name"] %></h3>
+      </div>
+
+      <div id="dob" class="basic-info">
+        <h2><%=@profile.keys[3].capitalize%></h2>
+        <h3 class ="profile-value"><%=@profile["dob"] %></h3>
+      </div>
+
+      <div id="phone" class="basic-info">
+        <h2><%=@profile.keys[13].capitalize%></h2>
+        <h3 class ="profile-value"><%=@profile["phone_number"] %></h3>
+      </div>
+    </div>
   </div>
 
-  <div id="problems" class="box">
-    <h3>Programming Problems</h3>
-    <a href="/primes">My Prime Number Solution</a>
+  <div class="section-three">
+    <div class="title">
+      <h2><%=@profile.keys[1].capitalize%></h2>
+        <div class="content">
+          <h3 class ="profile-value"><%=@profile["ambition"] %></h3>
+        </div>
+    </div>
   </div>
 
-  <div class="box">
-    <h3>Edit Profile</h3>
+  <div class="section-four">
+    <div class="title">
+      <h2>Codecademy</h2>
+        <div class="content">
+          <h3 class ="profile-value">Username: <%=@profile["codecademy"] %></h3>
+        </div>
+    </div>
+
+    <div class="info-container">
+      <div id="htmlcss" class="basic-info">
+        <h2>HTML & CSS </h2>
+        <h3 class ="profile-value"><%=@profile["web_track_completedness"] %></h3>
+      </div>
+
+      <div id="javascript" class="basic-info">
+        <h2>Javascript</h2>
+        <h3 class ="profile-value"><%=@profile["js_track_completedness"] %></h3>
+      </div>
 
-    <form action="/update" method="POST">
-      <fieldset>
-        <div class="form-group">
-          <label for="trainee[about]">About</label>
-          <input type="text" name="trainee[about]" value="<%= @profile["about"] %>"/>
+      <div id="ruby" class="basic-info">
+        <h2>Ruby</h2>
+        <h3 class ="profile-value"><%=@profile["ruby_track_completedness"] %></h3>
+      </div>
+    </div>
+  </div>
+
+<div class="section-five">
+    <div class="title">
+      <h2><%=@profile.keys[15].capitalize%></h2>
+        <div class="content">
+          <h3 class ="profile-value"><%=@profile["projects"] %></h3>
         </div>
+    </div>
+</div>
+
+  <div class="section-six">
+    <div class="box">
+      <div class="content">
+        <div class="title">
+          <h2>Edit Profile</h2>
+
+          <form action="/update" method="POST">
+            <fieldset>
+              <div class="form-group">
+                <label for="trainee[about]">About</label>
```

Comment:
Great stuff. You got this working perfectly! I'll be a nag again though (if you'll excuse me :wink:) and suggest there is some opportunity to reduce some of this duplication with something more dynamic. Remembering that iterating the `@profile` hash with its `key` and `value` attributes would be a good start...